torch/_decomp/decompositions.py

```diff
@@ -1901,8 +1901,8 @@ def is_same_size(a: Tensor, b: Tensor) -> bool:
     return a.shape == b.shape
 
 
-@register_decomposition(aten._reshape_alias)
-def _reshape_alias(x, shape, strides):
+@register_decomposition([aten._reshape_alias, aten._unsafe_view])
+def _reshape_alias(x, shape, *args):
```

Comment:
This will not respect the stride.
`aten.as_strided(x, shape, x.stride() if len(args) == 0 else args[0], x.storage_offset()` sounds better.

Reply:
So, my understanding is that `_reshape_alias` is just precomputing the strides that would be gotten from `view`, no?

Reply:
Quick check of the impl leads to `alias_with_sizes_and_strides` that sets the size and stride. So it definitely sets the given stride on the given Tensor while keeping the input's storage offset.

Reply:
Ok, I'll fix it in another PR (since the `_reshape_alias` decomp is pre-existing).